source/main/assets/tutorial/fr/welcome.md

```diff
@@ -0,0 +1,100 @@
+
+- [ ] travailler avec des fichiers et des répertoires,
+- [ ] apprendre les bases du Markdown,
+- [ ] installer Pandoc et LaTeX pour exporter mes fichiers.
+
+Certains éléments de l'éditeur sont interactifs, tout comme les cases à cocher. Une autre chose qui est très interactive, ce sont les tableaux. Il suffit de jeter un coup d'œil au tableau suivant : survolez-le avec votre souris pour voir apparaître quelques boutons qui vous permettent d'interagir avec !
+
+
+
+| Fichier                          | Objet                                                                         | Nom de fichier  |
+|----------------------------------|-------------------------------------------------------------------------------|-----------------|
+| Bienvenue sur Zettlr !           | Donne un aperçu rapide des capacités de Zettlr                                | welcome.md      |
+| Travailler avec les Zettelkästen | Introduction aux différentes caractéristiques des Zettelkästen dans Zettlr    | zettelkasten.md |
+| Les références avec Zettlr       | Souligne les potentialités du travail avec des bases de données de références | citing.md       |
+
+You can align columns as well as add and remove both rows and columns in the table. The table editor will always work on the column or row that is currently active. So, to remove a certain row, make sure that one cell within that row is selected. Feel free to play around a little bit with the table to get used to how it works!
+
+Dans le tableau, vous modifier l'alignement ou ajouter et supprimer des lignes et des colonnes. L'éditeur de tableau fonctionnera toujours sur la colonne ou la ligne active. Ainsi, pour supprimer une ligne, assurez-vous qu'une cellule de cette ligne est sélectionnée. N'hésitez pas à jouer un peu avec le tableau pour vous habituer à son maniement !
+
+## Ressources complémentaires 📚
+
+Vous y êtes arrivé ! La première partie de l'introduction est terminée. Nous n'avons pas abordé beaucoup de sujets ici, mais vous pouvez apprendre tout ce que vous voulez sur notre [documentation](https://docs.zettlr.com/) ! Ce qui pourrait vous intéresser, c'est [le puissant gestionnaire de balises](https://docs.zettlr.com/en/reference/settings/#manage-tags), ou les [options de recherche polyvalentes](https://docs.zettlr.com/en/core/search/).
+
+Mais maintenant que les bases sont suffisantes, nous sommes prêts à faire le grand saut ! Passons maintenant au tutoriel des Zettelkästen ! Pour y aller, il suffit de cliquer sur le lien wiki suivant : [[zettelkasten]]
+
+[^1] : Ce texte se trouve en bas de ce fichier. Mais, en fait, vous pouvez les mettre où vous voulez. Vous connaissez le meilleur ? Si vous cliquez sur cette note de bas de page tout en maintenant la touche `Cmd/Ctrl` enfoncée, vous pouvez changer le texte de place ! Essayez maintenant et enregistrez vos modifications en appuyant sur `Maj+Entrée`. Si vous voulez annuler vos modifications, cliquez simplement en dehors de la fenêtre popup !
```

Comment:
There's a space too much in here! I was already wondering if that's a bug that Zettlr did not recognize the footnote's text :D

Reply:
Sorry :) you're right. Pandoc style footnotes is `[^1]: bla bla` (without space :) )